dali/pipeline/operator/builtin/external_source.cc

```diff
@@ -86,11 +86,13 @@ and bandwidth.)code", false)
   .MakeInternal();
 
 DALI_SCHEMA(ExternalSource)
-  .DocStr(R"code(Allows externally provided data to be passed as an input to the pipeline,
-see :meth:`nvidia.dali.ops.ExternalSource.__init__`,
-see :meth:`nvidia.dali.fn.external_source`,
-:meth:`nvidia.dali.pipeline.Pipeline.feed_input` and
+  .DocStr(R"code(Allows externally provided data to be passed as an input to the pipeline.
+
+See :meth:`nvidia.dali.ops.ExternalSource.__init__`,
```

Comment:
This leads to strange link from fn API to the legacy one. Maybe we should use only fn here?